index.html

```diff
@@ -17,15 +17,71 @@ <h1>Solide State</h1>
             <a href="">Log in</a>
             <a href="">Sign Up</a>
         </nav>
+        </header>
```

Comment:
`header` is not below the opening tag. Accordingly change the other indentations.